### PR TITLE
f-cookie-banner@v0.20.0 - Reopen link should not be visible in ANZ

### PR DESCRIPTION
**Pull request by simonsandersje**
The reopen link was appearing on tenants using the legacy banner, which was not the correct behaviour

Also fixes the incorrectly wired useGreyBackground prop

### CODE REVIEW

File: packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue [hunk 1]
```diff
@@ -165,6 +166,13 @@ export default {
          */
         legacyBanner () {
             return this.shouldShowLegacyBanner === null ? this.config.displayLegacy : this.shouldShowLegacyBanner;
+        },
+        /**
+         * Check if the reopen cookie link should use a grey background
+         * @returns {Boolean}
+         */
+        shouldUseGreyBackground () {
```
Comment (DamianMullins) [inline packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue]: Do you need this completed property? Seems fine to pass along the prop as you were before if you don't need any additional logic?

Comment (DamianMullins) [reply packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue]: Ah, is it so you can configure it in Storybook?

**Review by DamianMullins — DISMISSED** [review]

**Review by minrwhite — DISMISSED** [review]
To stay aligned with how we are passing other props (e.g. `shouldShowLegacyBanner`), I'd prefer if `useGreyBackground` prop became `shouldUseGreyBackground` on the external interface, and the internal computed property was called `useGreyBackground`. That said, that would be a breaking change, and so I'm happy if this is done later.

**Comment by simonsandersje** [comment]
> To stay aligned with how we are passing other props (e.g. `shouldShowLegacyBanner`), I'd prefer if `useGreyBackground` prop became `shouldUseGreyBackground` on the external interface, and the internal computed property was called `useGreyBackground`. That said, that would be a breaking change, and so I'm happy if this is done later.

@benwhite-deltas Understood and changed - luckily, there are no consuming apps that use the prop yet

File: packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue (hunk 1 above)
Comment (dandel10n) [reply packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue]: Sorry, but I don't get the reason for this computed prop still

Comment (dandel10n) [reply packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue]: Can't you just use one `shouldUseGreyBackground` prop without a computed one?

Comment (simonsandersje) [reply packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue]: @dandel10n Computed value removed

**Review by minrwhite — APPROVED** [review]

**Review by FayeCarter — APPROVED** [review]

**Pull request merged**